zulia-client/src/test/java/io/zulia/client/WikiExamples.java

```diff
@@ -689,4 +691,32 @@ public void queryingWithMapper(ZuliaWorkPool zuliaWorkPool, Mapper<Article> mapp
 		List<Article> articles = searchResult.getMappedDocuments(mapper);
 	}
 
+	public void getNumericPercentiles(ZuliaWorkPool zuliaWorkPool) throws Exception {
+		List<Double> percentiles = new ArrayList<>() {{
+			add(0.0); // 0th percentile (min) - can be retrieved without percentiles
+			add(0.25); // 25th percentile
+			add(0.50); // median
+			add(0.75); // 75th percentile
+			add(1.0); // 100th percentile (max) - can be retrieved without percentiles
+		}};
+
+		Search search = new Search("myIndexName");
+		// Get the requested percentiles within 1% of their true value
+		search.addStat(new NumericStat("pubYear").setPercentiles(percentiles).setPercentilePrecision(0.01));
+		SearchResult searchResult = zuliaWorkPool.search(search);
+	}
+
+	public void numericFilters(ZuliaWorkPool zuliaWorkPool) throws Exception {
+		Search search = new Search("myIndexName");
+		search.addStat(new NumericStat("pubYear"));
+		// Search for pub years in range [2015, 2020]
+		search.addQuery(FilterFactory.rangeInt("pubYear").setMinValue(2015).setMinValue(2020).setEndpointBehavior(RangeBehavior.INCLUSIVE).toQuery());
```

Comment:
think this is supposed to say minValue / maxValue

Reply:
Thanks yeah it should